app/(app)/home/feed/index.tsx

```diff
@@ -15,7 +16,7 @@ export default function Page() {
   return (
     <MainContainer>
       <View>
-        <StyledText>Sheet</StyledText>
+        <FeedList />
```

Comment:
could you pass the feed data to the feedlist instead of fetching it in the `FeedList` component.

that way you can handle loading, empty data here
```suggestion
        <FeedList data={feed} />
```